sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/IVirtualMachineScaleSetVMsOperations.cs

```diff
@@ -629,7 +637,7 @@ public partial interface IVirtualMachineScaleSetVMsOperations
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<AzureOperationResponse> BeginDeleteWithHttpMessagesAsync(string resourceGroupName, string vmScaleSetName, string instanceId, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<AzureOperationResponse> BeginDeleteWithHttpMessagesAsync(string resourceGroupName, string vmScaleSetName, string instanceId, bool? forceDeletion = default(bool?), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
This one as well

Reply:
@markcowl  , added a customization, https://github.com/Azure/azure-sdk-for-net/blob/compute-2020-12/sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations/IVirtualMachineScaleSetVMsOperations.cs